staff/lluis-bagur/CV/index.html

```diff
@@ -0,0 +1,83 @@
+    <title>Document</title>
+    
+</head>
+<body>
+        <main>
+                <header>
+                  <h1>CV Lluis Bagur</h1>
+                  <nav style="">
+                        <ul>
+                                <button><a href="#education" title="index.html">Historial academico</a></button>
+                                <button><a href="#exp" title="index.html">Experiencia laboral</a></button>
+                                <button><a href="#aficiones" title="index.html">Aficiones</a></button>
+                        </ul>
+                </nav>
+                </header>
+                
+                <section>
+                    <h3>Datos personales:</h3>
+                    <p>Nombre: lluis Bagur</p>
+                    <p>Email: <a href="mailto:lluisbagur09@gmail.com">lluisbagur09@gmail.com</a></p>
+                    <p>Teléfono: 6XX-XXX-XXX</p>
+                </section>
+                <img src="./profile.jpg" class="profileImage" style="width: 200px">
+                <section id="education">
+                  <h2>Historial academico</h2>
+                  <details>
+                    <summary>EGB <time>1996 - 2001</time></summary>
+                        Educación basica, estudiada entre los 5 i 11 años en el colegio públigo "mare de deu del toro" 
+                  </details>
+                  <br>
+                  <details>
+                        <summary>ESO <time>2001 - 2006</time></summary>
+                            Educación secuntdaria obligatoria, estudiada entre los 11 i 16 años en el instituto público "IES J M Quadrado" 
+                      </details>
+                      <br>
+                  <details>
+                     <summary>FP GRADO SUPERIOR <time>2008 - 2010</time></summary>
+                          FP de grado superior de redes (ASIX), instituto público " IES Joan Ramis I Ramis" 
+                </details> 
+                <br>  
+                </section>
+
+                <section id="exp">
+                        <h2>Experiencia laboral</h2>
+                        <details>
+                          <summary>Técnico informático en SIM Informatica Menorca <time>2010 - 2011</time></summary>
+                              Primero en practicas y mas tarde como técnico, gestión y reparación de sistemas informaticos.
+                        </details><br>
+                        <details>
+                              <summary>Técnico informático en INNOVAGAMES Barcelona <time>2012 - 2013</time></summary>
+                              Técnico, gestión y reparación de sistemas informaticos y consolas.
+                        </details><br>
+                        <details>
+                           <summary>Múltiples trabajos no relacionados con el mundo de la informatica: <time>2006 - 2010 // 2013 - 2016</time></summary>
+                                <li>Ayudante recepción</li>
```

Comment:
I am not sure if it is a good practice to use the il without specifying an ol/ul (ordered/unordered lists)...